sdk/ai/azure-ai-inference/samples/sample_chat_completions_with_tracing.py

```diff
@@ -0,0 +1,191 @@
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+"""
+DESCRIPTION:
```

Comment:
Please udpate the description to mention that tracing is used...

Reply:
done